index.d.ts

```diff
@@ -15,6 +15,13 @@ declare namespace cpFile {
 		@default 0o777
 		*/
 		readonly directoryMode?: number;
+
+		/**
+		Working directory to find source files.
```

Comment:
```suggestion
		The working directory to find source files.
		
		The source and destination path are relative to this.
```

How about this? Just to make sure we also mention the destination path, which can also be relative to this.